test/GoldenTest.hs

```diff
@@ -114,7 +114,7 @@ goldenHDWallet = group $ do
             -> (XPub, XPrv, XSignature, Seed)
     runTest p (Mnemonic mw) pw ds (Root path) toSign =
         let -- 1. retrieve the seed
-            seed = cardanoSlSeed p mw
+            seed = fromMaybe undefined $ cardanoSlSeed p mw
```

Comment:
probably make more sense to use an error with something that is easy to grep than `undefined`